bindings/python/pydeck/pydeck/widget/widget.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+"""
+DeckGLWidget renders JSON into a deck.gl plot
```

Comment:
I think this could be a bit more articulate... ...accepts JSON encoded deck.gl layers and layer properties and renders them based on provided view states and properties.